devops/Makefile

```diff
@@ -60,7 +59,7 @@ define QA_DB_NAME
 endef
 
 define BUILD_TAG
```

Comment:
Maybe we can have the BUILD_TAG removed and replaced by the BUILD_REF. Thoughts?

Reply:
Yes we can, but most common usage was BUILD_TAG, rather using BUILD_REF.

Reply:
My point is, we can keep one. Right now we are just renaming BUILD_REF to BUILD_TAG. Maybe there is a benefit to doing it, or maybe I am missing something.

Reply:
yes, In our scenario `BUILD_TAG` and `BUILD_REF` is the same. @guru-aot @dheepak-aot @andrepestana-aot thought about removing `BUILD_TAG` ?